com.ibm.streamsx.topology/com.ibm.streamsx.topology.functional.python/PyFunctionHashRemover/PyFunctionHashRemover.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" ?>
+<operatorModel
```

Comment:
When I try to use spl.relational::Filter as the hash remover and set the output port schema to remove the hash attribute, I get the following compile error:
CDISP9083E ERROR: The schema at output port number 0 must match the schema at input port number 0.
Which seems to match the doc that says:
The Filter operator requires that the stream type of the output port matches the stream type of the input port. 
What am I missing?